scss/_toasts.scss

```diff
@@ -10,11 +10,11 @@
   box-shadow: $toast-box-shadow;
   @include border-radius($toast-border-radius);
 
-  &:not(.showing):not(.show) {
+  &.showing {
     opacity: 0;
   }
 
-  &.hide {
+  &:not(.show) {
```

Comment:
Can not `display: none` be used on the `.toast` class directly? :thinking: 

So this can be changed:
```css
&:not(.show) {
  display: none;
}
```

=>

```css
display: none;
```

Reply:
Good catch. :+1:

Reply:
tried it on https://github.com/twbs/bootstrap/pull/33610/commits/8a3a51366cb97c897e29607d5d44fde400394101 but seems more busy on changes